xsl/xccdf-guide-impl.xsl

```diff
@@ -411,6 +416,42 @@ Authors:
     </xsl:if>
 </xsl:template>
 
+<xsl:template name="table-of-contents-items">
+    <xsl:param name="item"/>
+    <xsl:param name="level"/>
+    <ol>
+        <xsl:for-each select="$item/cdf:Group">
+            <xsl:if test="cdf:title">
```

Comment:
In the rest of the guide and report we show ID when title is not available. I think we should do the same here instead of skipping that section.